epersons.md

```diff
@@ -74,3 +127,64 @@ the replace operation `[{ "op": "replace", "path": "/password", "value": "newpas
   "password": "newpassword",
 ```
 NOTE: The new password is currently returned after an update but this could be revisited later, see [#30]((https://github.com/DSpace/Rest7Contract/issues/30))
+
+## Create new EPerson
+
+**POST /api/eperson/epersons**
+
+To create a new EPerson, perform a post with the JSON below to the epersons endpoint when logged in as admin.
+
+```json
+{
+  "name": "user@institution.edu",
+  "metadata": {
+    "eperson.firstname": [
+      {
+        "value": "John",
+        "language": null,
+        "authority": "",
+        "confidence": -1
+      }
+    ],
+    "eperson.lastname": [
+      {
+        "value": "Doe",
+        "language": null,
+        "authority": "",
+        "confidence": -1
+      }
+    ]
+  },
+  "canLogIn": true,
+  "email": "user@institution.edu",
+  "requireCertificate": false,
+  "selfRegistered": true,
+  "type": "eperson"
+}
+```
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 422 Unprocessable Entity - if the email address was omitted or already exists
+
+## Linked entities
+### Groups
+**GET /api/eperson/epersons/<:uuid>/groups**
+
+A HAL link to retrieve the eperson groups of an eperson is included.
+This will return a pageable list of the groups this person is a direct member of
+
+> TODO: A solution to retrieve direct and indirect groups of an eperson is also required.
+> This would use GroupService.allMemberGroupsSet() and is used e.g. when viewing an EPerson as an admin
```

Comment:
This TODO is a bit odd here (as it talks about code changes).  But, if we don't have a better place to put this, we can leave it here temporarily.

Reply:
It doesn't require service or API code changes, but just states we'll need an additional rest endpoint to find all groups of an EPerson.
At this point in time I'd prefer not to include more endpoints in this PR because it's already a rather big PR for a contract.
The GroupService.allMemberGroupsSet can provide the functionality we need, but doesn't support paging

Reply:
That's fine. We'll just want to clean up this TODO statement at a later time then.